examples/UpdateContractMint/Program.cs

```diff
@@ -0,0 +1,260 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using CommandLine;
+using Concordium.Sdk.Client;
+using Concordium.Sdk.Types;
+using Concordium.Sdk.Wallets;
+
+namespace Transactions.UpdateContractMint;
+
+// We disable these warnings since CommandLine needs to set properties in options
+// but we don't want to give default values.
+#pragma warning disable CS8618
+
+internal sealed class Options
+{
+    [Option(
+        'k',
+        "keys",
+        HelpText = "Path to a file with contents that is in the Concordium browser wallet key export format.",
+        Required = true
+    )]
+    public string WalletKeysFile { get; set; }
+    [Option(HelpText = "URL representing the endpoint where the gRPC V2 API is served.",
+        Default = "https://grpc.testnet.concordium.com:20000/")]
+    public string Endpoint { get; set; }
+    [Option("contract", HelpText = "The index of the smart contract.", Default = "7936")]
+    public ulong Contract { get; set; }
+    [Option("token", HelpText = "Token ID to mint", Default = "00")]
+    public string TokenId { get; set; }
+    [Option("metadata-url", HelpText = "URL to the metadata", Default = "")]
+    public string MetadataUrl { get; set; }
+    [Option("max-energy", HelpText = "The maximum energy to spend on the module.", Default = 5000)]
+    public ulong MaxEnergy { get; set; }
+}
+
+/// <summary>
+/// Mint CIS-2 tokens for the 'cis2_multi' smart contract example.
+///
+/// https://github.com/Concordium/concordium-rust-smart-contracts/blob/86511efac8e335abac66176df895c21a5cde252c/examples/cis2-multi/src/lib.rs
+///
+/// This example demonstrates:
+///
+/// - Submitting a transaction updating a smart contract, where the parameter is constructed
+///   using a smart contract module schema.
+/// - Waiting for the transaction to finalize.
+/// - Checking the outcome and reading the logged events of the contract, parsing the events using
+///   a smart contract module schema.
+///
+/// The example assumes:
+/// - You have your account key information stored in the Concordium browser wallet key export
+///   format, and that a path pointing to it is supplied to it from the command line.
+/// - The 'cis2_multi' smart contract example is deployed on chain (already on testnet) with a
```

Comment:
I believe that another prerequisite is that the contract MUST be initialized - otherwise where should the contract address stem from.